health-services/project-factory/src/server/api/campaignApis.ts

```
@@ -1327,60 +1231,118 @@
 
+
+async function createBulkData(
+  request: any,
+  createAndSearchConfig: any,
+  params: any,
+  type: any,
+) {
+  const limit = createAndSearchConfig?.createBulkDetails?.limit;
+  const dataToCreate = request?.body?.dataToCreate;
+  const chunks = Math.ceil(dataToCreate.length / limit); // Calculate number of chunks
+  let creationTime = Date.now();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'creationTime' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)